docs/contribute.rst

```diff
@@ -64,13 +64,34 @@ sure that you have all required packages installed on your box::
 
     sudo dnf install gcc {python3,libvirt,krb5,libpq}-devel
 
+You may also need following packages if they are not already
+installed on your system::
+
+    sudo dnf install git python3-docutils
+
+In case you're using Centos Stream system you need to enable CRB
+repository first to make all the necessary packages available::
+
+    dnf config-manager --set-enabled crb
```

Comment:
It's called `powertools` on centos (see `podman run --rm -ti centos:stream8 dnf repolist all`)

All examples used `sudo`

Reply:
@lukaszachy Thanks for pointing that out :+1:  I currently only considered CentOS Stream 9 (I checked that via `tmt run provision --how virtual --image centos login`), but you're right - it is one of the [differences](https://centosfaq.org/centos/centos-9-stream-crb-repo/) between CentOS Stream 8 and 9. I will fix that and include notes for both cases.

Reply:
Ouch, so much for the consistency... I didn't realize centos9 has it different ;)

Reply:
At the end, there is no need for powertools repository. Both `python3-docutils` and `libvirt-devel` come from `appstream` repository in case of CentOS Stream 8. So CRB is only needed for the case of CentOS Stream 9.

Reply:
Done. All changes tested for CS 8 + 9.